apps/web/src/views/Info/utils/infoQueryHelpers.ts

```diff
@@ -18,7 +18,14 @@ export const multiQuery = async (
   let fetchedData = {}
   let allFound = false
   let skip = 0
-  const client = typeof endpoint === 'string' ? new GraphQLClient(endpoint) : endpoint
+  const client =
+    typeof endpoint === 'string'
+      ? new GraphQLClient(endpoint, {
+          headers: {
+            origin: 'https://pancakeswap.finance',
+          },
+        })
+      : endpoint
```

Comment:
Let's accept graphql client as part of the input parameters. And we can initialize gql client based on use cases. For example to use private endpoint on server end/build time, while public one on FE